Cargo.toml

```diff
@@ -24,159 +25,142 @@ aide = { version = "0", features = [
     "axum-extra",
     "axum-extra-query",
 ] }
-anyhow = "1.0"
-async-compression = { version = "0.3", features = [
+anyhow = "1"
+async-compression = { version = "0", features = [
     "futures-io",
     "gzip",
     "tokio",
     "zstd",
 ] }
-async-stripe = { version = "0.41", features = ["runtime-tokio-hyper"] }
-async-trait = "0.1"
-atty = "0.2"
-apache-avro = { version = "0.17.0", features = ["snappy"] }
+async-stripe = { version = "0", features = ["runtime-tokio-hyper"] }
+async-trait = "0"
+atty = "0"
+apache-avro = { version = "0", default-features = false }
 
-base64 = "0.13"
-bigdecimal = "0"
+base64 = "0"
+bigdecimal = { version = "0", default-features = false }
 bitvec = "1"
-bytecount = { version = "0.6.3", features = ["runtime-dispatch-simd"] }
-bytes = "1.6"
-bytesize = "1.1.0"
-bumpalo = { version = "3.14", features = ["collections"] }
-bytelines = "2.4"
-byteorder = "1.4"
-caseless = "0.2"
-chardetng = "0.1"
-chrono = { version = "0.4", features = ["serde"] }
+bytes = "1"
+bumpalo = { version = "3", features = ["collections"] }
+bytelines = "2"
+byteorder = "1"
+caseless = "0"
```

Comment:
It would be allowed for a library to break their api in each 0-version release, so I don't think it can ever be safe to support all 0 versions.